qiskit/utils/backend_utils.py

```diff
@@ -226,6 +226,9 @@ def is_simulator_backend(backend):
     backend_interface_version = _get_backend_interface_version(backend)
     if backend_interface_version <= 1:
         return backend.configuration().simulator
+    else:
+        if "simulator" in backend.name:
```

Comment:
My concern with this would the that the remote simulator "ibmq_qasm_simulator" would now be treated as local simulator wouldn't it. There is logic in run_circuits that dealt with max circuits around whether things were being run locally or not.

Reply:
I think that with the upcoming changes in runtime simulators we shouldn't worry about this too much.

Reply:
🙈